tox.ini

```diff
@@ -128,6 +121,10 @@ pip_pre =
     predeps: true
     !predeps: false
 
+
+[testenv:test]
```

Comment:
This is not needed 👍

Reply:
Where can we specify the test extras if not here?

Reply:
Has this been resolved, whether the extra `[testenv:test]` specification is really needed? `extras = test: test` is already generally defined above, and it does not seem to be required when running `tox r` locally.

Reply:
removed and still works in CI